packages/relay-service/lib/index.js

```diff
@@ -200,11 +205,20 @@ export async function resolveDidcommMessage({keyPairDocs, message}) {
 
   let result = jwe;
 
-  if (message.to) {
-    const keyPairDoc = keyPairDocs.find(doc => doc.controller === message.to);
+  let recipients = jwe?.recipients?.map(recipient => recipient.header.kid);
+
+  if (recipients) {
```

Comment:
is it valid to have a message with no recipients?

Reply:
if it's not a didcomm message, yes. Maybe I should rename this variables, its being used to check if didcomm.decrypt must be used or not

Reply:
let me update that

Reply:
done, i've pushed an update. This case applies for jwt message, which is not encrypted using didcomm, but should be a valid message for us